tests/e2e_regression/expected/spec.body

```diff
@@ -283,6 +283,20 @@ paths:
           schema:
             type: integer
           description: A filter on transaction status code.
+        - in: query
+          name: after
+          schema:
+            type: string
+            format: date-time
+          description: A filter on minimum transaction time, inclusive.
+          example: *iso_timestamp_1
+        - in: query
+          name: before
+          schema:
+            type: string
+            format: date-time
+          description: A filter on maximum transaction time, inclusive.
```

Comment:
I see we already have the "inclusive" part for blocks filtering (both consensus and runtime). How do you feel about changing it to "exclusive", just for the `before` arg, both for blocks and txs?

The web UI won't particularly care because it can specify the timestamps with arbitrary precision. But for other clients (and e.g. manual use), it feel very unintuitive to say `before=2022-03-15`, read that `before` is inclusive, and still get no results for `2022-03-15`. That will happen because `2022-03-15` expands to `2022-03-15T00:00:00`.

Reply:
Agreed, will update, thanks!